i18n/en_AU.json

```diff
@@ -4348,7 +4348,7 @@
   "pricing_modal.addons.USSupport": "United States-only based support",
   "pricing_modal.addons.dedicatedDB": "Dedicated database",
   "pricing_modal.addons.dedicatedDeployment": "Dedicated virtual secure cloud deployment (Cloud)",
-  "pricing_modal.addons.dedicatedEncryption": "Dedicated encryption keys 99%",
+  "pricing_modal.addons.dedicatedEncryption": "Dedicated encryption keys",
```

Comment:
Even though we likely know what this should be, we should only change `en.json`. IIRC trying to change other languages/dialects we know can confuse the downstream translator process.

Reply:
Alright. Reverting...